nh_observations/frequencies.py

```diff
@@ -67,23 +62,32 @@
         EVERY_8_HOURS[0]: EVERY_8_HOURS,
         EVERY_12_HOURS[0]: EVERY_12_HOURS,
         EVERY_DAY[0]: EVERY_DAY,
-        EVERY_3_DAYS[0]: EVERY_DAY,
-        EVERY_WEEK[0]: EVERY_DAY
+        EVERY_3_DAYS[0]: EVERY_3_DAYS,
+        EVERY_WEEK[0]: EVERY_WEEK
     },
     'Medium': {
-        # EVERY_15_MINUTES[0]: EVERY_15_MINUTES,
-        # EVERY_30_MINUTES[0]: EVERY_30_MINUTES,
         EVERY_HOUR[0]: EVERY_HOUR
     },
     'High': {
-        # EVERY_15_MINUTES[0]: EVERY_15_MINUTES,
         EVERY_30_MINUTES[0]: EVERY_30_MINUTES
     },
     'Unknown': EVERY_4_HOURS,
     'Transfer': EVERY_15_MINUTES,
     'Obs Restart': EVERY_HOUR
 }
 
+# To summarise the mappings below, frequencies after refusals are capped at
+# 24 hours.
+#
+# Could have just made a simple function but decided to write them out long
+# hand as they may change in the future. Return the tuple with the label rather
+# than just the time so it can be useful in more scenarios.
+PATIENT_REFUSAL_ADJUSTMENTS = dict.copy(FREQUENCIES_BY_RISK)
```

Comment:
It was in fact a shallow copy. Good catch @Gimpneek, would've almost definitely caused bugs in the future.